benchmarks/mo_create_study.py

```diff
@@ -0,0 +1,44 @@
+import json
+import sys
+
+from kurobako import solver
+from kurobako.solver.optuna import OptunaSolverFactory
+
+import optuna
+
+
+optuna.logging.disable_default_handler()
+
+
+def create_study(seed: int) -> optuna.Study:
+    seed
```

Comment:
Adding a  comment above this line is useful to understand.
```suggestion
    # Avoid the fail by `flake8`.
    seed
```

Reply:
LGTM. I add this comment.